.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+src/js/*.config.js
```

Comment:
I mean, we can fix it specifically later. Or rename. Or move it to a secret folder? I don't quite know convention. Happy to fix it later.